markdown.tmLanguage.base.yaml

```diff
@@ -138,8 +138,8 @@ repository:
         (<?)(\S+?)(>?)      # The url
         [ \t]*          # Optional whitespace
         (?:
-            ((\().+?(\)))    # Match title in quotes…
-          | ((").+?("))    # or in parens.
+            ((\().+?(\)))    # Match title in parens…
+          | ((["']).+?(["']))    # or in quotes.
```

Comment:
Won't this also match titles whose start and end quotes don't match? For example `"bad title'`

Reply:
Yes, I noticed this (since I've referenced textmate/markdown.tmbundle#52) and have a fix incoming for this.